cmd/operator/config.json

```diff
@@ -6,5 +6,6 @@
 	"PhaseDuration": 3,
 	"ResponseDuration": 1,
 	"PowDepth": 0,
-	"Port": 8080
-}
\ No newline at end of file
+	"Port": 8080,
+	"Honest": true
```

Comment:
Rename to something more specific, like "RespondChallenges". Hones is quite vague and can mean many things, like not sending proofs etc. When we later add other protocol-deviation behavior, it helps naming the things more specifically.

Reply:
Renamed it to `RespondChallenges`.